pkg/jobs/registry.go

```diff
@@ -895,6 +899,141 @@ func (r *Registry) Start(ctx context.Context, stopper *stop.Stopper) error {
 	})
 }
 
+func (r *Registry) createJobsStatsPollerIfNotExists(ctx context.Context) error {
+	if r.knobs.DisableJobsMetricsPolling {
+		return nil
+	}
+
+	for {
+		const TxnName = "Jobs Stats Poller Txn"
+		txn := r.db.NewTxn(ctx, TxnName)
```

Comment:
why not use r.db.Txn(func(....)) and have function lookup + create if
needed?

Reply:
Ack. It uses `r.db.Txn` now.